third_party/3/orjson.pyi

```diff
@@ -2,16 +2,23 @@ from typing import Any, Callable, Optional, Union
 
 __version__ = str
 
-def dumps(__obj: Any, default: Optional[Callable[[Any], Any]] = ..., option: Optional[int] = ...,) -> bytes: ...
+def dumps(
```

Comment:
This part will probably fail CI because we use a different Black configuration than the orjson repo.